Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -194,14 +194,12 @@ If not installing with Helm, here are the steps to install with manifests.
 Create the object store resources:
 
 1. Create the [external object store CR](https://github.com/rook/rook/blob/master/deploy/examples/object-external.yaml) to configure connection to external gateways.
-2. Create an [Object store user](https://github.com/rook/rook/blob/master/deploy/examples/object-user.yaml) for credentials to access the S3 endpoint.
```

Comment:
This normal rgw-user for customers to deploy in the environment. This is not `radosgw-adminops` user. We support access external rgw via cephobjectstoreuser similar to OBC. So please keep this change

Reply:
@thotz I got your point so we can have separate section of it either use obc or objectuser

Reply:
We need both options for the user, if we necessary split them into two separate session

Reply:
updated